mvn-extension/src/test/java/org/arquillian/smart/testing/mvn/ext/dependencies/SurefireApiDependencyTest.java

```diff
@@ -75,7 +76,7 @@ public void should_autocorrect_strategy()
         Model model = prepareModelWithSurefirePlugin("2.20");
         final Configuration conf = configureWithAutocorrect("nwe");
         final DependencyResolver dependencyResolver = new DependencyResolver(conf);
-        model.addDependency(new DependencyResolver.SurefireApiDependency("2.19.1"));
+        model.addDependency(new DependencyResolver.SurefireApiDependency("2.20"));
```

Comment:
why this change?

Reply:
Well it is a mocked instance, and you can change it without any worries, it works with any number there.

Reply:
so, again, why you have changed it? is there any reason for it?

Reply:
Well because the model is prepared with 2.20, so to make it more real the dependency should be 2.20 as well isn't it?See line 76

Reply:
Yeah, I missed that line :-). Thx
anyway, I'm not sure if adding the surefire dependency is really necessary here... But that can be part of another PR ;-)